app/bolt_listeners.py

```diff
@@ -53,6 +53,7 @@ def just_ack(ack: Ack):
 )
 DEFAULT_LOADING_TEXT = ":hourglass_flowing_sand: Wait a second, please ..."
 
+BOT_ID = 'U05RC3T79SS'
```

Comment:
I found if I hardcode the bot ID, then we can determine which role to assign for communicating with model